src/lib.rs

```diff
@@ -69,7 +69,7 @@ pub fn generate_config(path: &str) -> Result<(), Box<dyn Error>> {
     let mut times = Vec::new();
     let mut start_sec = 0;
     for _ in 0..length {
-        times.push(format!("{}:{}", start_sec / 60, start_sec % 60));
+        times.push(format!("{:02}:{:02}", start_sec / 60, start_sec % 60));
```

Comment:
Why is this ```:02``` and not ```:00```?

Reply:
Using `:02` will left-pad the number with zeroes to a 2 character limit (e.g. 8 will become 08 and 12 will just stay 12)